nyuoss/ninja-shadowdash-nyu
# Team 2 - Converter

Remade the PR because rebasing commits somehow introduced additional commits into the PR when changes were done that were suggested by the other team's review.
The review by the other team is in the old PR: https://github.com/nyuoss/ninja-shadowdash-nyu/pull/18

Directory structure of files modified in this PR:
```
- README.md # readme for the converter
- build.sh # builds the converter binary in debug-build/ folder with the name shadowdash
- src/ # contains the converter source code, previously contained ninja source code
- converter/
	- include/
		- manifest.h # contains definitions for shadowdash namespace that are required to compile the output of the converter into a library
	- testing/
		- example.build.ninja
		- example.zlib.ninja
	- run-tests.sh
```

For Reviewer:
- See README.md for how to build, run and test the converter.
- See the src/ directory for changes that were done to Ninja to make it behave as a converter.
- See the converter/ directory for test harness (run-tests.sh) and testcases.
